hibernate-core/src/main/javadoc/package.html

```diff
@@ -33,7 +33,7 @@ <h3>Extensions</h3>
     <li>{@link org.hibernate.Transaction} / {@link org.hibernate.transaction.TransactionFactory}</li>
     <li>{@link org.hibernate.context.CurrentSessionContext}</li>
     <li>{@link org.hibernate.dialect.Dialect}</li>
-    <li>{@link org.hibernate.dialect.resolver.DialectResolver}</li>
+    <li>{@link org.hibernate.orm.test.dialect.resolver.DialectResolver}</li>
```

Comment:
Looks like this happened by accident.

```suggestion
    <li>{@link org.hibernate.dialect.resolver.DialectResolver}</li>
```

Reply:
Strange.  Thanks for the fix